charms/kfp-persistence/src/pebble_components.py

```diff
@@ -0,0 +1,77 @@
+import dataclasses
+import logging
+from typing import Dict
+
+from charmed_kubeflow_chisme.components.pebble_component import PebbleServiceComponent
+from ops.pebble import Layer
+
+logger = logging.getLogger(__name__)
+
+
+@dataclasses.dataclass
+class PesistenceAgentServiceConfig:
+    """Defines configuration for PersistenceAgent Service."""
+
+    KFP_API_SERVICE_NAME: str
+    NAMESPACE: str
+
+
+class PersistenceAgentContainer(PebbleServiceComponent):
+    # TODO: Should this be something we subclass to define settings, or should
```

Comment:
To this question (and if I understood it correctly), are pebble layers for our charms that standard? I don't have the knowledge to answer this, but if yes, this would make sense to create e.g. `command`, `environment` inside the charm's code and then only pass those as arguments.